setup.py

```diff
@@ -24,6 +24,7 @@
         "pillow",
         "imagecodecs",
         "openslide-python",
+        "large-image[all]<1.9",
```

Comment:
I'm curious why the version is pinned.

Reply:
The code broke when tensorflow released 2.7.0; so I fixed 'tensorflow<2.7.0' to get bug fixes but not the newer versions.  I did the same for other dependencies just in case.  So, it's not a very strong reason.  Should I remove the version limit for large_image?

Reply:
I don't usually pin versions unless there is an actual issue.  In general, any minor update *should* keep backwards compatibility. On other packages we've had specific point release fail (e.g., !=4.2.1 ends up being the version specifier, as just that one release is bad).

If you do want to pin without a specific reason to do so, the usual motif is to allow upgrades until the next major version.  In Python, this is specified via `~=` (so `~=1.8` is equivalent to `>=1.8,<2`).

Reply:
I have removed the version pin for large-image with the latest push.